ports/vcpkg-gn/vcpkg.json

```
@@ -1,8 +1,7 @@
 {
   "name": "vcpkg-gn",
-  "version-date": "2021-11-16",
-  "port-version": 3,
-  "description": "https://gn.googlesource.com/gn/+/4aa9bdfa05b688c58d3d7d3e496f3f18cbb3d89e",
+  "version-date": "2024-02-22",
+  "description": "https://gn.googlesource.com/gn/+/03d10f1657b4ddace618c34ab61b6357d1ae9c9a",
```

```suggestion
  "description": "GN is a meta-build system that generates build files for Ninja (GN is the build system for Chromium, Fuchsia, and related projects)",
```

and make the documentation point to https://gn.googlesource.com/gn/+/main/docs/reference.md instead of vcpkg.io

---

Instead of vcpkg.io?